MyString.java

```diff
@@ -19,12 +19,37 @@ public static void main(String[] args) {
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
         // Replace the following statement with your code
-        return null;
+        String result = "";
+        for(int i = 0; i < str.length(); i++) {
+            char c =str.charAt(i); 
+                if (c >= 65 && c <= 90) {
+                    c += 32;
```

Comment:
1. use chars 'A', 'Z'
2. Sometimes +32 is not clear enough, use `(char)(c - 'A' + 'a')`